Plugins/Public/base_plugin/BuildModule.cpp

```diff
@@ -28,12 +28,15 @@ wstring BuildModule::GetInfo(bool xml)
 			uint good = i->first;
 			uint quantity = i->second;
 
+			if (quantity <= 0) { continue; }
```

Comment:
Please do not do one-liner if statements. Also an uint cannot be negative, so it's just a zero check, simply do
`if(!quantity)
{
continue;
}`